Makefile

```diff
@@ -195,7 +195,7 @@ pylint:
 format_quick:
 	@echo "Quick format on ONlY changed/new files"
 	@$(FIND_MOD_NEW) | xargs -r clang-format -i -style=file
-	@$(FIND_MOD_NEW_PY) | xargs autopep8 $(AUTOPEP8_CONFIG) -i
+	@$(FIND_MOD_NEW_PY) | xargs -r autopep8 $(AUTOPEP8_CONFIG) -i
```

Comment:
if no python files were changed, this make command would fail (and you wouldn't be able to commit). The -r flag is added so the following commands run if there isn't any empty input from the pipe

Reply:
Thanks for fixing this!